aws-rds-cfn-common/src/main/java/software/amazon/rds/common/logging/LogRuleSet.java

```diff
@@ -0,0 +1,46 @@
+package software.amazon.rds.common.logging;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.function.Consumer;
+
+import lombok.AllArgsConstructor;
+
+public interface LogRuleSet {
+    void handle(Object object);
```

Comment:
NIT: understanding the context of the LogRuleSet domain, I would propose 2 alternative names for this method: `accept` or `consume`. This class is still effectively a consumer.

Reply:
Done.